internal/api/shared/schemas/reloader.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schemas
+
+import (
+	"context"
+
+	"github.com/perses/common/async"
+	"github.com/perses/perses/internal/api/impl/v1/dashboard/validator"
+	"github.com/perses/perses/internal/config"
+	log "github.com/sirupsen/logrus"
+)
+
+type reloader struct {
+	async.SimpleTask
+	validator   validator.Validator
+	schemasPath string
```

Comment:
I wonder if it's really useful to store schemasPath here and have it as an argument of LoadSchemas(), given that the validator has already the info ?